### PR TITLE
StateVisualizer: Add protected access to some internal members to allow subclassing to add new states

### LINKED ISSUE #925: [FEATURE REQUEST] on the state visualizer to add in a hover state for theming when the button is Toggled ON
## Describe the problem
currently when a button is toggled there are not visuals to know if you are hovering over the button

## Describe the solution you'd like
I would love to be able to have all visuals states available for our toggle buttons, similar to when it's a regular button

### PR DESCRIPTION
**Pull request by johntensuan**
StateVisualizer: Add protected access to some internal members to allow subclassing to add new states

resolves #925

### CODE REVIEW

**Review by keveleigh — APPROVED** [review]
Seems reasonable to me! Thank you for contributing!

File: org.mixedrealitytoolkit.uxcore/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "org.mixedrealitytoolkit.uxcore",
-  "version": "3.2.2-development",
+  "version": "3.3.2-development",
```
[the remark this answers is not in the document]
Comment (johntensuan) [reply org.mixedrealitytoolkit.uxcore/package.json]: Thanks for catching that!

**Comment by johntensuan** [comment]
@keveleigh Can you help me complete the merge for this PR? Thanks!

**Comment by keveleigh** [comment]
> @keveleigh Can you help me complete the merge for this PR? Thanks!

Done! Thank you!

**Pull request merged**